### PR TITLE
Hotfix/build imports issue fix

### PR DESCRIPTION
**Pull request by bigbitbus**
# Description
Explicitly import modules when building opta binary since dynamic imports are not detected by pyinstaller.

# Safety checklist
* [X ] This change is backwards compatible and safe to apply by existing users
* [ X] This change will NOT lead to data loss
* [ X] This change will NOT lead to downtime who already has an env/service setup

## How has this change been tested, beside unit tests?
Manually

### CODE REVIEW

File: opta.spec [hunk 1]
```diff
@@ -4,7 +4,37 @@ from PyInstaller.utils.hooks import collect_data_files
 
 block_cipher = None
 
-
+himps = [
```
Comment (RemyDeWolf) [inline opta.spec:7]: can we sort this list for maintenance?
also it can be broken into 3 lists for each cloud providers

Comment (NitinAgg) [reply opta.spec:7]: Can we instead have a function which generates this list

Comment (bigbitbus) [reply opta.spec]: @RemyDeWolf @NitinAgg great suggestions! Addressed both  in new commit.

**Comment by bigbitbus** [comment]
Passing tests here - https://app.circleci.com/pipelines/github/run-x/opta/514/workflows/293fb6de-d40d-46e8-9382-b485669a5d0b

Ignore the redis test - its the expiring aws eks credentials issue https://github.com/hashicorp/terraform/issues/29182

File: opta.spec [hunk 2]
```diff
@@ -1,9 +1,53 @@
...
+        modname = filename.split(".py")[0]
+        himps.append(".".join(["modules", modname, modname]))
+  return sorted(himps)
+
+
 
 block_cipher = None
 
+himps = generate_module_import_array()
+# himps = [
```
Comment (NitinAgg) [inline opta.spec:19]: nit: remove these comments?

**Review by NitinAgg — APPROVED** [review]

File: opta.spec (hunk 2 above)
Comment (bigbitbus) [reply opta.spec]: Done

**Pull request merged**